public/components/GeoPlot.vue

```diff
@@ -232,8 +234,32 @@ export default Vue.extend({
     mapZoom(): number {
       return routeGetters.getGeoZoom(this.$store);
     },
+
     rowSelection(): RowSelection {
       return routeGetters.getDecodedRowSelection(this.$store);
+    },
+
+    isSatelliteImages(): boolean {
```

Comment:
I create a simple `boolean` to know if the dataset contains satellite images or not.
Maybe this will be done by a primitives?

Reply:
We're figuring out the best way to do this now.  There are two options: it gets flagged in the `Dataset` data that comes back from the `distil/datasets/<name>` route, so you would be able to pull out of the `store/datasets/` vuex module, or it will be something to check in the returned `Task` data that we fetch via `distil/task/<dataset>/<target>/<var-names>` and store in `store/datasets` as well.  Either way, you won't have to worry about fetching the data from there server, you'll just need to update this function to check the store for one of the cases above.